src/AmsterdamPHP/Bundle/MeetupBundle/Service/EventService.php

```diff
@@ -130,4 +130,24 @@ public function getPastEvents($onlyMonthlyMeeting = false)
 
         return $events;
     }
+
+    /**
+     * @return MultiResultResponse
+     */
+    public function getCurrentMonthlyMeeting()
+    {
+        //Get Upcoming events
+        $events = $this->api->getEvents(
+            [
+                'group_urlname' => $this->group,
+                'status'        => 'upcoming',
+                'text_format'   => 'plain',
+                'time'          => '0m,1m'
+            ]
+        );
+
+        return $events->filter(function($event) {
+            return strpos($event['name'], 'Monthly Meeting') !== false;
```

Comment:
Are they always exactly matching this string `Monthly Meeting`? Might want to do a `stripos` just to be sure?